internal/worker/rundynamic.go

```diff
@@ -14,35 +14,85 @@ import (
 	"github.com/ossf/package-analysis/internal/pkgmanager"
 	"github.com/ossf/package-analysis/internal/sandbox"
 	"github.com/ossf/package-analysis/pkg/api/analysisrun"
+	"github.com/ossf/package-analysis/pkg/api/pkgecosystem"
 )
 
-var nonSpaceControlChars = regexp.MustCompile("[\x00-\x08\x0b-\x1f\x7f]")
-
 /*
-RunDynamicAnalysis runs dynamic analysis on the given package in the sandbox
-provided, across all phases (e.g. import, install) valid in the package ecosystem.
-Status and errors are logged to stdout. There are 4 return values:
+DynamicAnalysisResult holds all the results from RunDynamicAnalysis
 
-DynamicAnalysisResults: Map of each successfully run phase to a summary of
+Data: Map of each successfully run phase to a summary of
```

Comment:
Is there a better name than "data" that could be used here?

"Data" is not particularly descriptive of what is actually being stored.

Reply:
How about `AnalysisData`?

Reply:
`AnalysisData` is better